BaragonUI/app/actions/api/services.es6

```diff
@@ -29,3 +31,20 @@ export const ReloadService = buildJsonApiAction(
     url: `/state/${serviceId}/reload`,
   })
 );
+
+export const RemoveUpstreams = buildJsonApiAction(
```

Comment:
we should make sure to add this action to the reducers as well, the returned output will be more easily accessible